pynbody/snapshot/ramses.py

```diff
@@ -1046,6 +1048,30 @@ def _not_cosmological(self):
 
         return not_cosmological
 
+    @property
+    def has_negative_iords(self):
+        return self._has_negative_iords()
+
+    def _has_negative_iords(self):
+        return self.dm['iord'].min() < 0
+
+    @property
+    def has_potential_negative_iords_bug(self):
+        return self._has_potential_negative_iords_bug()
+
+    def _has_potential_negative_iords_bug(self):
+        if self.negative_iords_on_purpose:
+            return False
+
+        if self.is_cosmological and self.has_negative_iords:
+            warnings.warn("This Ramses snapshot is cosmological and has DM iords that are negative. "
```

Comment:
This warning seems to duplicate the one in the bridge. I don't think it's right for this to warn here - after all, someone might just want to know the value of `has_potential_negative_iords_bug` without issuing any warning at all.